x-pack/platform/plugins/private/data_visualizer/public/application/file_data_visualizer/file_data_visualizer.tsx

```diff
@@ -40,23 +40,73 @@ export const FileDataVisualizer: FC<Props> = ({
     fileUpload,
     fieldFormats,
   };
-
   const EmptyContext: FC<PropsWithChildren<unknown>> = ({ children }) => <>{children}</>;
   const CloudContext = cloud?.CloudContextProvider || EmptyContext;
 
+  const autoAddInference = undefined;
+  const autoCreateDataView = true;
+  const indexSettings = undefined;
```

Comment:
Are these variables being updated anywhere? Can they be moved out as a static constant?